### PR TITLE
Epa eds

### PR DESCRIPTION
**Pull request by rizzom5000**

### CODE REVIEW

**Comment by natoverse** [comment]
Where is a location that has pesticide data? I zoomed around a bit and only saw the "No data" box.

**Comment by rizzom5000** [comment]
We can only get data through 2009 unfortunately. I considered removing the option of selecting 2010+. Maybe it would be better to default to 2009.

**Comment by rizzom5000** [comment]
I'll also add that to the notes below the map.

**Comment by natoverse** [comment]
Yeah, that's better. One other quirk: starting zoomed out, the counties are points, and clicking them shows the data. Zooming in, they turn into color-coded polygons, which looks much nicer but also does not work for clicking. I think the ideal would be to start a little more zoomed in so the counties are polygons, but then fix the clicking so it shows the data. Is that possible?

**Comment by rizzom5000** [comment]
I've set the default zoom to 6, so the map now defaults to zoomed over NE and IA. I've also commented out the code that adds the state polygon, so all of the county data should be available. 

I think we can find a way to add this back in in a future version if we refactor the logic so that it is added before the county layer is added.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
-[![Dependency Status](https://david-dm.org/atsid/usda-challenge.svg)](https://david-dm.org/atsid/usda-challenge)
-[![devDependency Status](https://david-dm.org/atsid/usda-challenge/dev-status.svg)](https://david-dm.org/atsid/usda-challenge#info=devDependencies)
-[![Dependency Status](https://gemnasium.com/atsid/usda-challenge.svg)](https://gemnasium.com/atsid/usda-challenge)
-
-# usda-challenge
-ATS + EchoUser submission for USDA Innovation Challenge
+# EPA EDS-RFI Response
+Forked from ATS + EchoUser submission for USDA Innovation Challenge
```
Comment (stopyoukid) [inline README.md]: Might be nice to have the Github URL

**Comment by stopyoukid** [comment]
:shipit:

**Pull request merged**